join.go

```diff
@@ -85,7 +85,8 @@ func JoinHorizontal(pos Position, strs ...string) string {
 			b.WriteString(block[i])
 
 			// Also make lines the same length
-			b.WriteString(strings.Repeat(" ", maxWidths[j]-ansi.PrintableRuneWidth(block[i])))
+			// b.WriteString(strings.Repeat(" ", maxWidths[j]-ansi.StringWidth(block[i])))
```

Comment:
What is the reasoning for this? Confused about the TODO as well. Is this just to remind yourself to update it?

Reply:
Hey @mikelorant, thank you for reviewing this PR. This indeed was just to remind me to update the line I was testing. I just updated the branch to uncomment this line.

Reply:
👍